src/web/components/Core/Tooltip.scss

```diff
@@ -5,9 +5,17 @@
 }
 
 .tooltip-content {
-  color: var(--theme-info-content);
-  font-family: 'Inter';
+  color: var(--theme-button-text);
+  border-radius: 4px;
+  background-color: var(--theme-background-sidepanel);
+  padding: 10px;
   font-weight: 400;
-  font-size: 1rem;
-  line-height: 1.18rem;
+  font-size: 10px;
```

Comment:
Font size and line height should be in rem - we want them to scale if the user zooms or has a larger font set (e.g. if they have impaired vision and need larger text).